CHANGELOG.md

```diff
@@ -20,346 +20,416 @@ See [`eslint-plugin-babel`'s releases](https://github.com/babel/eslint-plugin-ba
 
 <!-- DO NOT CHANGE THESE COMMENTS - See .github/actions/trigger-github-release/update-changelog.js -->
```

Comment:
It looks like an auto-formatter accidentally formatted this file, could you revert it?